tests/test_utilities.py

```diff
@@ -156,6 +156,159 @@ def test_check_all_contiguous():
+    DX -= DX.mean()
+    DY = DX
+    for dy in DY:
+        for dx in DX:
+            offset = galsim.PositionD(dx,dy)
+            offset_list.append(offset)
+
+    np.random.seed(42) # for generating the same random permutation everytime
+    rand_idx = np.random.permutation(len(offset_list))
+    im_list_randperm = [im_list[idx] for idx in rand_idx]
+    offset_list_randperm = [offset_list[idx] for idx in rand_idx]
+    # Input to N as a tuple
+    img_randperm = galsim.utilities.interleaveImages(im_list_randperm,(n,n),offsets=offset_list_randperm)
+
+    np.testing.assert_array_equal(img_randperm.array,img.array,\
+        err_msg="Interleaved images do not match when 'offsets' is supplied")
+    assert img_randperm.scale == img.scale
+
+    # 1c) Catching errors if offsets
+    offset_list = []
+    im_list = []
+    n = 5
+    DX = np.array([-0.67,-0.33,0.,0.33,0.67])
+    DY = DX
+    for dy in DY:
+        for dx in DX:
+           offset = galsim.PositionD(dx,dy)
+           offset_list.append(offset)
+           im = galsim.Image(16,16)
+           gal.drawImage(image=im,offset=offset,scale=1.0,method='no_pixel')
+           im_list.append(im)
+
+    VE = ValueError('No error')
+    try:
+        N = (n,n)
+        galsim.utilities.interleaveImages(im_list,N=N,offsets=offset_list,catch_offset_errors=True)
+    except ValueError as VE:
+        pass
+    message =  "'offsets' must be a list of galsim.PositionD instances with x values"\
+                          +" spaced by 1/{0} and y values by 1/{1} around 0 for N = ".format(n,n)+str(N)
+    assert VE.message == message
+
+    offset_list = []
+    im_list = []
+    n = 5
+    DX = np.arange(0.,1.,1./n)
+    DY = DX
+    for dy in DY:
+        for dx in DX:
+            offset = galsim.PositionD(dx,dy)
+            offset_list.append(offset)
+            im = galsim.Image(16,16)
+            gal.drawImage(image=im,offset=offset,scale=1.0,method='no_pixel')
+            im_list.append(im)
+
+    VE = ValueError('No error')
+    try:
+        N = (n,n)
+        galsim.utilities.interleaveImages(im_list,N=N,offsets=offset_list,catch_offset_errors=True)
+    except ValueError as VE:
+        pass
+    assert VE.message == message
+
+    # 2a) Increase resolution along one direction - square to rectangular images
+    n = 2
+    g = galsim.Gaussian(sigma=3.7,flux=100.)
+    g1 = g.shear(g=1.*(n**2-1)/(n**2+1),beta=0.0*galsim.radians)
+    gal1 = g1 #galsim.Convolve([g1,galsim.Pixel(1.0)])
+    im_list = []
+    offset_list = []
+  
+    # Generating offsets in a natural way
+    DY = np.arange(0.0,1.0,1.0/(n*n))
+    DY -= DY.mean()
+    for dy in DY:
+        im = galsim.Image(16,16)
+        offset = galsim.PositionD(0.0,dy)
+        offset_list.append(offset)
+        gal1.drawImage(im,offset=offset,scale=1.0,method='no_pixel')
+        im_list.append(im)
+
+    img = galsim.utilities.interleaveImages(im_list,N=[1,n**2],offsets=offset_list,add_flux=False,suppress_warnings=True)
```

Comment:
Should add a comment highlighting your test of `add_flux=False` here.  The flux in the test profile below has the same flux as the input profiles.  Unlike the flux for the other tests where you scaled it up.